applications/IOIOConnectionTesterAndroid/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.android.application'
 android {
     compileSdkVersion 30
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 21
```

Comment:
Currently this app is without this issue, because here no tests are running

Reply:
I realize that, but using different minSdkVersions in different apps would cause some confusion to people testing these apps on their hardware, in that one would work and others would fail. If you disagree, I can change these back to 14.

Reply:
yes please, change it back when we can